src/Microsoft.AspNetCore.Localization/QueryStringRequestCultureProvider.cs

```diff
@@ -35,6 +35,7 @@ public override Task<ProviderCultureResult> DetermineProviderCultureResult(HttpC
             }
 
             var request = httpContext.Request;
+
```

Comment:
@ryanbrandenburg I saw in almost the aspnet repos - including localization one - adding blank line is required after `var` line
@dougbu you mean I should remove the blank lines before conditional block?